### PR TITLE
testing(newsletter, contact): fixup failing async specs

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
See #636 has failing specs.

## What is the new behavior?
This replaces the failing specs with a mocked version of the service with `jasmine-marbles`.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: libs/contact/src/driver/hubspot/contact.service.spec.ts [hunk 1]
```diff
@@ -1,64 +1,45 @@
...
-            HttpClient,
-            DOCUMENT, 
-            Router, 
-            Title, 
-            DaffContactConfigToken
-          ],
+					provide: DaffHubspotFormsService,
+          useValue: {
+            submit(): Observable<any> {
+              return hot('--a', { a: {test: "123"}});
```
Comment (griest024) [inline libs/contact/src/driver/hubspot/contact.service.spec.ts]: double quotes should be single

File: libs/newsletter/src/driver/hubspot/newsletter.service.spec.ts [hunk 2]
```diff
@@ -1,62 +1,44 @@
...
-          ],
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			providers: [
+				DaffNewsletterHubspotService,
+				{
+					provide: DaffHubspotFormsService,
+          useValue: {
+            submit(): Observable<any> {
+              return hot('--a', { a: {test: "123"}});
```
Comment (griest024) [inline libs/newsletter/src/driver/hubspot/newsletter.service.spec.ts]: double quotes should be single

File: libs/contact/src/driver/hubspot/contact.service.spec.ts (hunk 1 above)
Comment (damienwebdev) [reply libs/contact/src/driver/hubspot/contact.service.spec.ts]: Fixed

File: libs/newsletter/src/driver/hubspot/newsletter.service.spec.ts (hunk 2 above)
Comment (damienwebdev) [reply libs/newsletter/src/driver/hubspot/newsletter.service.spec.ts]: Fixed

**Pull request merged**